mele-manoa/mele-manoa
# Issue 1 - Landing Page

Steps to test:
- Boot up the application
- Verify that all the images show up correctly
- Verify that the buttons have a hover animation
- Verify that no extra links show up on the navbar when user is not logged in

Note: Landing page when user is logged in is still a work in progress
